packages/core/src/components/table/_table.scss

```diff
@@ -122,32 +142,20 @@ table.#{$ns}-html-table {
   }
 
   &.#{$ns}-interactive {
-    tbody tr:hover td {
-      background-color: rgba($gray5, 0.4);
-      cursor: pointer;
+    tbody tr {
+      &:hover td {
+        background-color: rgba($gray5, 0.3);
+        cursor: pointer;
+      }
+
+      &:active td {
+        background-color: rgba($gray5, 0.4);
```

Comment:
might we already have variables we can use here? something like default minimal buttons?

Reply:
I doubt it cause the color / opacity is a function of the size of the element (this one is big!).

but you have a point anyway, we have these in buttons, tags, and probably more places. if anything is done, it belongs to follow up PR